node/cli/src/cli.rs

```diff
@@ -29,6 +29,10 @@ pub struct RunCmd {
     /// Force using Vara native runtime.
     #[arg(long = "force-vara")]
     pub force_vara: bool,
+
+    /// The upper limit for the amount of gas a validator can burn in one block.
+    #[arg(long, short = 'g')]
```

Comment:
such short variant isn't intuitive for me

Reply:
`-g` for "gas". Also unlikely to conflict with any other args should there be any in the future.

Other suggestions?

Reply:
or we can disable the short variant completely: `--max-gas` is short enough